### PR TITLE
feat: area-features with its dedicated block

### LINKED ISSUE #32: Implement feature icons in the info section
The info section of the vip area is the section after the hero component. 
In some pages there are 3 icons shown, e.g. https://www.realmadrid.com/area-vip/partido-vip/silver-club-1anfiteatro

Screenshot 2023-05-03 at 17 11 56

Screenshot 2023-05-03 at 17 12 18

The first 2 icons "Atención personalizada" and "Catering Gourmet" open a popup with information when the user clicks on it. This can be implemented as a fragments.

The third one opens a popup with an iframe with the 360 view. The url of this iframe needs to be configured in the document itself.

### PR DESCRIPTION
**Pull request by vdua**
Fix #32

Test URLs:
- Before: https://main--realmadrid--hlxsites.hlx.page/drafts/vdua/baloncesto-vip1
- After: https://32-area-features--realmadrid--hlxsites.hlx.page/drafts/vdua/baloncesto-vip1

### CODE REVIEW

**Comment by vdua** [comment]
Seems like iframe needs to be added only with intersection observer to maintain LH Score.

**Comment by vdua** [comment]
Pending items
 * close on escape key
 * close button in the popup
 * ensure 100 LHS

File: blocks/area-features/area-features.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+      }
+      observer.unobserve(li);
+    }
+  });
+}
+
+function enableIframeLoad(li) {
+  const options = {
+    rootMargin: '0px',
+    threshold: 0.5,
```
Comment (vdua) [inline blocks/area-features/area-features.js:44]: Have no experience working with intersection observer. Let me know if the threshold value needs to be modified

**Review by chicharr — APPROVED** [review]
@vdua it looks good to me.

**Pull request merged**